### ISSUE TITLE
SVG Sprites with nested tags

### ISSUE THREAD

**Issue by Br33b0h** [FortAwesome/Font-Awesome#12976]
### Describe the problem
We have configured FontAwesome to nest `<svg>` tags. Using SVG Sprites with this configuration runs into an error `h[0].attributes.class is undefined` and the `<i>` tag has no nested `<svg>` tag.

If configuration is changed to replace `<svg>` tags everything works fine. Also if data-fa-symbol attribute is removed from the `<i>` tag no error occurs.

### What did you expect?
No error and nested `<svg>` tag inside the `<i>` tag

### What version and implementation are you using?

Version: 5.0.10
Browser and version: all

- [x] SVG with JS
- [ ] Web Fonts with CSS
- [ ] SVG Sprites
- [ ] On the Desktop

### Reproducible test case
[Example on JSFiddle](https://jsfiddle.net/xgo311nc/)
```
<!doctype html>
<html>
  <head>
    <script defer src="https://use.fontawesome.com/releases/v5.0.10/js/all.js" integrity="sha384-slN8GvtUJGnv6ca26v8EzVaR9DC58QEwsIk9q1QXdCU8Yu8ck/tL/5szYlBbqmS+" crossorigin="anonymous"></script>
    <script>
      FontAwesomeConfig = {autoReplaceSvg: "nest"};
    </script>
  </head>
  <body>
    <i class="fas fa-minus-circle" data-fa-symbol></i>
    <div>
      <svg class="svg-inline--fa fa-lg"><use xlink:href="#fas-fa-minus-circle"></use></svg>
    </div>
  </body>
</html>
```

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

As per [documentation](https://fontawesome.com/how-to-use/svg-with-js#auto-replace-svg-nest), you just need `<i class="fas fa-minus-circle"></i>`

Why are you using `data-fa-symbol`?

https://jsfiddle.net/xgo311nc/1/

**Comment by Br33b0h** [comment]
We use `data-fa-symbol` because we have many repititions of the same icon. Above example was just simplified to demonstrate the behaviour with SVG sprites and nested tags. :)

**Comment by tagliala** [comment]
Got it, thanks. I can confirm there is an issue between nesting tags and svg symbols features

https://fontawesome.com/how-to-use/svg-with-js#auto-replace-svg-nest
https://fontawesome.com/how-to-use/performance-and-security#symbols-using-svg-sprites

@robmadole 

this is a fiddle showing the issue with the example reported in the documentation: https://jsfiddle.net/tagliala/xgo311nc/4/

If we remove `autoReplaceSvg: "nest"` icons will show up

**Comment by robmadole** [comment]
Great! Nice bug report. We'll get this in the queue.

**Comment by tcastelli** [comment]
Any input on this? Just faced the prolem one year later and Haven't found any workaround

**Comment by robmadole** [comment]
Ok. I think we've got this fixed in 6.0.0-beta3. We don't have plans to backport this to 5 and will be asking everyone to upgrade.
